Wind-River/cloud-platform-deployment-manager
# Update OAM Network using oam API

This commits enable the OAM network using the
oam network API instead of the addresspool API.

Test Plan:
[Day-1]
1. Perform a subcloud installation with a modified deployment-config.yaml configuring the OAM Network with a different floatinAddress.
2. Check the OAM Network was not updated.

[Day-2]
1. After system installation setup the platformNetwork with new values of the OAM Network.
2. Apply the deployment configuration
3. Check the OAM Network was updated
- system oam-show
- system addrpool-list
